src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview/RequestOverview.tsx

```diff
@@ -0,0 +1,335 @@
+  return (
+    <VStack align="flex-start">
+      <Text textStyle="subhead-2" textColor="text.label" noOfLines={1}>
+        {name}
+      </Text>
+      <Breadcrumb
+        as={Text}
+        textOverflow="ellipsis"
+        overflowX="auto"
+        spacing="2px"
+        separator={<BiChevronRight color={theme.colors.text.helper} />}
+      >
+        {path.map((item) => {
+          return (
+            <BreadcrumbItem>
+              <Text textStyle="caption-2" textColor="text.helper">
+                {item}
+              </Text>
+            </BreadcrumbItem>
+          )
+        })}
+      </Breadcrumb>
+    </VStack>
+  )
+}
+
+type LastEditedMetaProps = Pick<
+  EditedItemProps,
+  "lastEditedBy" | "lastEditedTime"
+>
+const LastEditedMeta = ({
+  lastEditedBy,
+  lastEditedTime,
+}: LastEditedMetaProps): JSX.Element => {
+  const { date, time } = getDateTimeFromUnixTime(lastEditedTime)
+  return (
+    <HStack>
+      <Avatar
+        bg="primary.100"
+        name={lastEditedBy.slice(0, 2).split("").join(" ")}
+        textStyle="caption-2"
+        textColor="primary.400"
+        size="sm"
+      />
+      <VStack align="flex-start">
+        <Text textStyle="caption-2" textColor="text.helper">
+          {date}
+        </Text>
+        <Text textStyle="caption-2" textColor="text.helper">
+          {time}
+        </Text>
+      </VStack>
+    </HStack>
+  )
+}
+
+export interface RequestOverviewProps {
+  items: EditedItemProps[]
+}
+
+export const RequestOverview = ({
+  items,
+}: RequestOverviewProps): JSX.Element => {
+  const {
+    isExpanded,
+    inputRef,
+    handleExpansion,
+    handleCollapse,
+  } = useSearchbar({})
+  const columnHelper = createColumnHelper<EditedItemProps>()
+
+  const columns = [
+    columnHelper.display({
+      id: "filter",
+      header: () => (
+        <Th w="3rem">
+          <IconButton
+            aria-label="sort by file type"
+            icon={<BiFilterAlt />}
+            variant="link"
+          />
+        </Th>
+      ),
+      cell: (props) => getIcon(props.row.original.type),
+    }),
+    columnHelper.accessor((row) => `${row.name} ${row.path}`, {
+      id: "itemName",
+      cell: ({ row }) => (
+        <ItemName name={row.original.name} path={row.original.path} />
+      ),
+      header: () => (
+        <Th textTransform="capitalize">
+          <Text textStyle="subhead-2" textColor="text.label">
+            Item name
+          </Text>
+        </Th>
+      ),
+    }),
+    columnHelper.accessor(
+      (row) => `${row.lastEditedBy} ${row.lastEditedTime}`,
+      {
+        id: "lastEdited",
+        header: () => (
+          <Th textTransform="capitalize" w="12.5rem">
+            <HStack spacing="0.25rem">
+              <Text textStyle="subhead-2" textColor="text.label">
+                Last edited
+              </Text>
+              <IconButton
+                icon={<BiSort fontSize="1rem" />}
+                aria-label="view file changes"
```

Comment:
I think this label does not match the intention of the button.

Reply:
yep, will change!

Reply:
changed to `sort by file type`